dataminer-overview/Training_courses_and_certification/Overview_Training_certification.md

```diff
@@ -14,3 +14,101 @@ For example, the DataMiner Fundamentals Certification is intended for DataMiner
+
+### Certification process
+
+As soon as the ordering process is finalized and your certification process can be started, you will receive an automated email from Skyline Academy with information about the process and the deadline.
+
+You will also receive an email with your user ID and password to access the training DMA in order to practice and to take the exam.
+
+To book an appointment, you will need to use the booking link that you will find in the email you received with information about the start of the certification process.
+
+> [!NOTE]
+>
+> - The certification process is valid for 3 months, unless otherwise agreed.
+> - An appointment can be canceled or rescheduled up to 96 hours before it starts. If you cannot update the appointment online, please contact Skyline Academy or your team of examiners.
+> - If you ordered two or more certifications, the Configurator certification process will only start after you have completed the Fundamentals exam.
+
+### Exam format and grading
+
+The exam consist of an online verbal exam with one of our official DataMiner trainers, using Microsoft Teams, as well as (optional) offline assignments. The maximum duration of the verbal exam is 1 hour. To pass the exam, you need to score at least 85%.
+
+The examiner will use a predefined exam plan to ask different questions per module of the training. The questions vary from strictly theoretical questions to showing things on the training DMA.
+
+> [!NOTE]
+>
+> - Turn on your camera during the exam.
+> - Exams are conducted in English.
+> - By default, the training DMA will be used to conduct the exam.
+> - You are allowed to use DataMiner Docs during the exam.
+> - If you do not pass the exam, you have one free retake.
+> - Exams are recorded to ensure the integrity of the process and to provide a reference for review if needed.
+
+### Preparation tips
+
+To prepare effectively for the exam:
+
+- Study materials: review the [recommended course](https://community.dataminer.services/courses/dataminer-administrator/) and [online resources](https://docs.dataminer.services/).
+- Practice questions: use the [quiz](https://community.dataminer.services/courses/dataminer-administrator/quizzes/dataminer-administrator/) at the end of the course to practice. The quiz randomly selects 10 questions out of a pool of questions.
+- Schedule check-ins with your examiner via the booking page if you have any questions.
+
+### Examples questions
+
+- What technology is used to manage and execute backup schedules?
+- Is it possible to test queries within Cube? If so, where can you find this feature?
+- How would you create a sample scheduled task with specific details such as name, schedule, time of execution, and actions?
+- How do you define the relationship between objects in Microsoft Visio with DataMiner shapes for elements, views, etc.?
```

Comment:
This seems a bit odd. Is this what you mean?
```suggestion
- How do you define the relationship between shapes in Microsoft Visio and DataMiner objects such as elements, views, etc.?
```

Reply:
Will need to check with one of the trainers. 
The questions  they use in the exam plan is:
How do you define the relationship between objects in MS Visio with DataMiner shapes for elements, views, etc....

Answer: 
You use shape data in visio. The labels and value fields need to be used.

Reply:
If that is the answer, I think the way I rephrased it is indeed better, because strictly speaking there is no such thing as "DataMiner shapes", but instead there are Visio shapes referring to DataMiner objects.

Reply:
thanks - I've updated the question in the exam plan as well